hazelcast/src/main/java/com/hazelcast/monitor/TimedMemberState.java

```diff
@@ -143,6 +154,7 @@ public void fromJson(JsonObject json) {
         final JsonObject jsonMemberState = getObject(json, "memberState");
         memberState = new MemberStateImpl();
         memberState.fromJson(jsonMemberState);
+        sslEnabled = getBoolean(json, "sslEnabled");
```

Comment:
I think it's better to use  the overloaded version of `getBoolean` that takes a default value.